frappe/utils/boilerplate.py

```diff
@@ -98,7 +98,13 @@ def make_boilerplate(dest, app_name):
 	with open(os.path.join(dest, hooks.app_name, hooks.app_name, "config", "docs.py"), "w") as f:
 		f.write(frappe.as_unicode(docs_template.format(**hooks)))
 
-	print("'{app}' created at {path}".format(app=app_name, path=os.path.join(dest, app_name)))
+	# initialize git
+	app_directory = os.path.join(dest, hooks.app_name)
+	frappe.commands.popen('git init', cwd=app_directory)
+	frappe.commands.popen('git add .', cwd=app_directory)
+	frappe.commands.popen('git commit -m "Initialize app"', cwd=app_directory)
```

Comment:
we have [`gitpython`](https://github.com/frappe/frappe/blob/develop/requirements.txt#L16) installed as a dependency, we could use that instead of using subprocess. maybe also can ask the user if they want to initialize the repository, and ask for a commit message for it. how about something like this:

```suggestion
	app_repo = git.Repo.init(app_directory)
	app_repo.git.add(u=True)
	app_repo.index.commit("feat: Initialized Frappe App")
```

this is all considering that `import git` is called in the file

Reply:
@Thunderbottom Using `gitpython` now, thanks for the suggestion. However, I don't think we should complicate this by giving more options. Most frameworks that generate app boilerplates just initialize the repo automatically. And it is not very difficult to amend the commit message.